### PR TITLE
Feature/issue #85

### PR DESCRIPTION
**Pull request by tteguayco**
A ```chromedriver.exe``` that matches the version of the installed Google Chrome browser on the user's machine can be now automatically downloaded.

* The installed version of Google Chrome is queried via Registry: https://stackoverflow.com/a/14300356
* Based on it, we can get the matching version of ChromeDriver we have to install: https://chromedriver.chromium.org/downloads/version-selection

This Pull Request contains an addition to the README that shows how this new feature can be used.

Tested "on the fly", as I couldn't think of a unit test that makes sense (it depends on the machine which the project runs on).

Issue requesting feature: https://github.com/rosolko/WebDriverManager.Net/issues/85

### CODE REVIEW

**Comment by Piedone** [comment]
Looks great, thank you! I'll check this out this week.

File: WebDriverManager/DriverConfigs/Impl/ChromeConfig.cs [hunk 1]
```diff
@@ -63,5 +66,25 @@ public virtual string GetLatestVersion()
...
+                using (var content = response.GetResponseStream())
+                {
+                    if (content == null) throw new ArgumentNullException($"Can't get content from URL: {uri}");
+                    using (var reader = new StreamReader(content))
+                    {
+                        var version = reader.ReadToEnd().Trim();
+                        return version;
+                    }
+                }
+            }
```
Comment (Piedone) [inline WebDriverManager/DriverConfigs/Impl/ChromeConfig.cs:87]: You can merge these `using`s (omit the brackets of the first one) or if OK in this project also use the new `using var` notation.

File: WebDriverManager/DriverConfigs/Impl/FirefoxConfig.cs [hunk 2]
```diff
@@ -46,6 +47,11 @@ public virtual string GetLatestVersion()
             }
         }
 
+        public virtual string GetMatchingBrowserInstalledVersion()
+        {
+            throw new NotImplementedException();
```
Comment (Piedone) [inline WebDriverManager/DriverConfigs/Impl/FirefoxConfig.cs:52]: Hmm, I awaited this to be thrown when using FF with `version: "matching_installed"` but it didn't. What could be the issue?

File: WebDriverManager/DriverConfigs/Impl/ChromeConfig.cs (hunk 1 above)
Comment (rosolko) [reply WebDriverManager/DriverConfigs/Impl/ChromeConfig.cs]: In my opinion current implementation is pretty simple and obvious

**Comment by rosolko** [comment]
@tteguayco @Piedone I've updated code a little bit, can you please review it?

File: WebDriverManager/DriverConfigs/Impl/ChromeConfig.cs (hunk 1 above)
Comment (tteguayco) [reply WebDriverManager/DriverConfigs/Impl/ChromeConfig.cs]: I'll let you guys decide this issue. I grabbed already existing code to implement this part.

**Comment by tteguayco** [comment]
> @tteguayco @Piedone I've updated code a little bit, can you please review it?

Looks great to me, @rosolko. Your naming convention is much better and clear. 

Could you check what @Piedone says about the exception ```NotImplementedException```` not being thrown when using a browser that does not support this new feature?

**Comment by Piedone** [comment]
Looks great! I've also tested it and works as it should. I now also get the exception for Firefox.

**Pull request merged**